python/pyspark/ml/tests.py

```diff
@@ -1027,6 +1027,21 @@ def test_storage_levels(self):
         self.assertEqual(als.getFinalStorageLevel(), "DISK_ONLY")
         self.assertEqual(als._java_obj.getFinalStorageLevel(), "DISK_ONLY")
 
+    def test_unknown_strategy(self):
```

Comment:
I see a similar test was added for the storage level params, but I don't exactly understand why we need these. We don't implement these types of tests for other classes and params and this is just checking the getters and setters. Also, the `fit` call seems unnecessary.